core/src/banking_stage/leader_slot_timing_metrics.rs

```diff
@@ -244,6 +244,9 @@ pub(crate) struct ProcessPacketsTimings {
     // Time spent running the cost model in processing transactions before executing
     // transactions
     pub cost_model_us: u64,
+
+    // banking stage processing time histogram for sampled packets
+    pub process_sampled_packets_us_hist: histogram::Histogram,
```

Comment:
This will almost certainly have between 0-2 counts per block on mnb, meaning it will probably be so noisy as to be useless.

Reply:
I think we care much more about time from sigverify to banking stage picking up the packet from channel.

Reply:
The sampling mechanism goal was to sample the system without tracking everything. Even a couple of data points per slot over long time time can still provide insight into where the time is spent over various stage. Time to scheduler is good stats to have. I will defer to future PRs to reduce this change set.